test/new-e2e/tests/remote-config/rc_ssl_config_test.go

```diff
@@ -26,6 +27,7 @@ type sslConfigSuite struct {
 var sslMismatchConfig string
 
 func TestSslConfigSuite(t *testing.T) {
+	t.Parallel()
```

Comment:
As more tests are run in parallel, did you check we don't reach any EC2 limits when a lot of PRs are merged?

Reply:
That's a good point. I think we're still safe for now as we're not very close from the limits. We should get alerts as soon as we dangerously approach the limit.
https://app.datadoghq.com/s/yB5yjZ/x8t-pk3-nq8